### PR TITLE
Pancakes!

### PR DESCRIPTION
**Pull request by nauticall**
# About the pull request

Adds pancakes. Simple as.

Can be made with 3 dough slices and 5 milk. A bit expensive, but you're rewarded with three pancakes! Three!

# Explain why it's good for the game
Who doesn't like pancakes?

No, but seriously, more food items are good. This was yet another one of the results of black magic I had done to circumvent impossibilities. Pancakes for days.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

image
image

</details>

# Changelog 

:cl:
add: Adds pancakes.
/:cl:

### CODE REVIEW

**Review by sleepynecrons — APPROVED** [review]
sprites 🥞👍

File: code/game/objects/items/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -953,6 +953,19 @@
...
+	desc = "Golden brown creamy pancakes fresh from the griddle. Drizzled with maple syrup and topped with a slice of butter."
+	icon_state = "pancakes"
+	filling_color = "#bb9857"
+
+/obj/item/reagent_container/food/snacks/pancakes/Initialize()
+	. = ..()
+	reagents.add_reagent("bread", 6)
+	reagents.add_reagent("sugar", 4)
+	reagents.add_reagent("milk", 5)
+	bitesize = 2
```
Comment (Zonespace27) [inline code/game/objects/items/reagent_containers/food/snacks.dm:967]: Why is this set on init

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/reagent_containers/food/snacks.dm (hunk 1 above)
Comment (Zonespace27) [reply code/game/objects/items/reagent_containers/food/snacks.dm:967]: That shouldn't be how its set then

Comment (nauticall) [reply code/game/objects/items/reagent_containers/food/snacks.dm:967]: sorry but this is just how it's done, rewriting the entire bit of food code for one pancake PR is waaaay out of scope for this

Comment (Zonespace27) [reply code/game/objects/items/reagent_containers/food/snacks.dm:967]: i mean, you can literally just put `bitesize = 2` in the type declaration and that fixes it

**Pull request merged**